docs/dictionary/function/machine.lcdoc

```diff
@@ -30,22 +31,24 @@ The <machine> <function> <return|returns> a <string>.
 -   iPhone Simulator: the device is a simulated iPhone
 -   iPad: the device is the iPad
 -   iPad Simulator: the device is a simulator iPad
-
+-   : macOS, Unix and Android returns described below
```

Comment:
FM will be the bitness of the version of FM which is launched which on win can either be 32-bit or 64-bit.